src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js

```diff
@@ -140,13 +155,69 @@ const CourseSessionStatus = () => {
         setHighlightSession((prevHighlight) => !prevHighlight);
     }, []);
 
+    const studentEmail = enrollmentData?.enrollment.student.user.email;
```

Comment:
Move the unenroll logic into the "Unenroll" button. Pull + Merge development to get these changes